acceptance/bundle/templates/default-python/classic/script

```diff
@@ -6,6 +6,7 @@ trace $CLI bundle validate -t prod
 
 # Do not affect this repository's git behaviour #2318
 mv .gitignore out.gitignore
+rm -fr .databricks
```

Comment:
For consistency this should also only remove the `.gitignore` file.

The different approaches in different templates is fine, but if they diverge it should include a comment why it is done so we remain intentional about every bit of behavior in these tests.

Reply:
done in c2536d542f47e9ee3b0be0fb7ea08a24501c47b0